modules/serverless-installing-cli-linux.adoc

```diff
@@ -6,19 +6,31 @@
 [id="installing-cli-linux_{context}"]
 = Installing the Knative CLI for Linux
 
-For Linux distributions, you can download the `kn` CLI directly as a `tar.gz` archive.
+If you are using a Linux distribution that does not have RPM or another package manager installed, you can install the Knative (`kn`) CLI as a binary file. To do this, you must download and unpack a `tar.gz` archive and add the binary to a directory on your `PATH`.
 
-// no prereqs?
+.Prerequisites
+
+* If you are not using RHEL or Fedora, ensure that *libc* is installed in a directory on your library path.
++
+[IMPORTANT]
+====
+If *libc* is not available, you might see the following error when you run CLI commands:
+
+[source,terminal]
+----
+$ kn: No such file or directory
+----
+====
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/kn-linux-amd64.tar.gz[`kn` CLI].
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/kn-linux-amd64.tar.gz[Knative (`kn`) CLI `tar.gz` archive].
```

Comment:
With the latest release of Serverless (1.21) we started providing the GPG signature of files. I think, we should advise our users to verify the downloaded file with it. 

See: https://issues.redhat.com/browse/SRVCLI-316?focusedCommentId=20133090&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20133090

Reply:
Out of scope for this PR - please raise a Jira to cover this update @cardil

Reply:
This probably should've been included in release notes as well, or may need to be added in other places in the docs